src/IoTSettings.ts

```diff
@@ -3,47 +3,36 @@
 
 import * as path from 'path';
 import * as vscode from 'vscode';
-import * as sdk from 'vscode-iot-device-cube-sdk';
 
 import {CancelOperationError} from './CancelOperationError';
 import {ConfigHandler} from './configHandler';
-import {OSPlatform} from './constants';
+import {ConfigKey, OSPlatform} from './constants';
 import {PickWithData} from './Models/Interfaces/UI';
+import {getHomeDir, getPlatform} from './utils';
 
 export class IoTWorkbenchSettings {
-  private _workbenchPath = '';
+  private workbenchPath = '';
+  private static instance: IoTWorkbenchSettings|undefined;
 
   private constructor() {}
 
-  static async createAsync() {
-    const iotWorkbenchSettings = new IoTWorkbenchSettings();
-
-    iotWorkbenchSettings._workbenchPath =
-        await IoTWorkbenchSettings.getWorkbenchPath();
+  static async getInstance() {

```

Comment:
You could add return type in method signature as other method.

Reply:
Fixed.